src/platforms/android/index.mdx

```diff
@@ -31,7 +31,7 @@ android {
 // ADD SENTRY ANDROID AS A DEPENDENCY
 dependencies {
     // https://github.com/getsentry/sentry-android/releases
-    implementation 'io.sentry:sentry-android:{version}'
+    implementation 'io.sentry:sentry-android:{{ packages.version('sentry.android', '{version}') }}'
```

Comment:
@bruno-garcia fyi made an assumption on what the package name will be called, same w/ sentry.java below